generators/typescript/model/type-reference-example-generator/src/GeneratedTypeReferenceExampleImpl.ts

```diff
@@ -133,6 +138,16 @@ export class GeneratedTypeReferenceExampleImpl implements GeneratedTypeReference
                 return `"${primitiveExample.datetime.toISOString()}"`;
             case "date":
                 return `"${primitiveExample.date}"`;
+            case "uint":
+                return primitiveExample.uint;
+            case "uint64":
+                return primitiveExample.uint64;
+            case "float":
+                return primitiveExample.float;
+            case "bigInteger":
+                return primitiveExample.bigInteger;
```

Comment:
Based on the `string` representation, I think we'll need:

```suggestion
                return `"${primitiveExample.bigInteger}"`;
```